rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -146,5 +147,26 @@ describe('DSL', function () {
         await lens!.click();
         await driver.wait(async () => (await editor.getTextAtLine(9)).trim() === "b := 2;", 20_000, "a variable should be changed to b");
     });
+
+    it("quick fix works", async() => {
+        const editor = await ide.openModule(TestWorkspace.picoFile);
+        await editor.setTextAtLine(9, "az := 2;");
+        await editor.moveCursor(9,3);                   // it's where the undeclared variable `az` is
+        await ide.hasErrorSquiggly(editor, Delays.verySlow);   // just make sure there is indeed something to fix
+        const editorView = new EditorView();
+
+        // find an editor action button by title
+        const button = await editorView.getAction("Change to a");
+
+        if (button) {
```

Comment:
how about instead of return true/false, we use driver.wait on the `getAction` call, so that it will retry it untill the action exists, and then if it doesn't appear in time, we have an error that says: hey we couldn't find the `Change to a` action.

Reply:
sure. I think you have this "in your fingers"? Can you refactor pls.

Reply:
done, see eb6fb747

Reply:
nice!

Reply:
1) DSL
       quick fix works:
     TimeoutError: We should get the change to a code action
Wait timed out after 40221ms
      at /home/runner/actions-runner/_work/rascal-language-servers/rascal-language-servers/rascal-vscode-extension/node_modules/selenium-webdriver/lib/webdriver.js:915:22
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Reply:
This is still the case; right after the rewrite and continuing into the other fixes I've made.